File: https:/github.com/lucius-lt/leetcode.git/33-SearchInRotatedSortedArray/33-SearchInRotatedSortedArray.c
// Last updated: 7/16/2026, 1:12:28 AM
int search(int* nums, int numsSize, int target) {
    int a=0;
    for(int i =0;i<numsSize;i++)
    {
        if(target ==nums[i])
        {
         return i;
        }
    }
    
    return -1;
}

File: https:/github.com/lucius-lt/leetcode.git/33-SearchInRotatedSortedArray/33-SearchInRotatedSortedArray.java
// Last updated: 7/16/2026, 1:12:20 AM
class Solution {
    public int search(int[] nums, int target) {
        for(int i =0;i<nums.length;i++){
            if(nums[i]==target) return i;
        }
        return -1;
    }
}